### PR TITLE
moving from gsutil to gcloud in e2e tests

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
Moving from gsutil to gcloud in e2e tests as gsutil will anyways getting deprecated and in TPC environment gsutil is not supported.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Louhi pipeline on all the VMs
2. Unit tests - NA
3. Integration tests - Automated

### CODE REVIEW

File: tools/integration_tests/util/operations/file_operations.go [hunk 1]
```diff
@@ -477,7 +477,7 @@ func UploadGcsObject(localPath, gcsObjPath string, uploadGzipEncoded bool) error
 // Uses 'gsutil rm gs://gcsObjPath'
 // Alternative 'gcloud storage rm gs://gcsObjPath' but it doesn't work on kokoro VM.
 func DeleteGcsObject(gcsObjPath string) error {
-	_, err := ExecuteGsutilCommandf("rm gs://%s", gcsObjPath)
+	_, err := ExecuteGcloudCommandf("rm -r gs://%s", gcsObjPath)
```
Comment (ashmeenkaur) [inline tools/integration_tests/util/operations/file_operations.go:480]: recursive not required here

File: tools/integration_tests/managed_folders/admin_permissions_test.go [hunk 2]
```diff
@@ -51,17 +51,21 @@ var (
...
 
 func (s *managedFoldersAdminPermission) Setup(t *testing.T) {
 	createDirectoryStructureForNonEmptyManagedFolders(t)
 }
 
 func (s *managedFoldersAdminPermission) Teardown(t *testing.T) {
-	// The 'gsutil rm -rf' command doesn't work on managed folders.
-	// We'll clean up the test directory but leave managed folders.
-	setup.CleanupDirectoryOnGCS(path.Join(bucket, testDir))
+	// Due to bucket view permissions, it prevents cleaning resources outside of managed folders. So we are cleaning only managed folders resources only.
```
Comment (ashmeenkaur) [inline tools/integration_tests/managed_folders/admin_permissions_test.go:62]:
```suggestion
	// Due to bucket view permissions, it prevents cleaning resources outside of managed folders. So we are cleaning managed folders resources only.
```

File: tools/integration_tests/util/setup/setup.go [hunk 3]
```diff
@@ -362,7 +362,7 @@ func CleanUpDir(directoryPath string) {
 
 	for _, d := range dir {
 		err := os.RemoveAll(path.Join([]string{directoryPath, d.Name()}...))
-		if err != nil {
+		if err != nil && !strings.Contains(err.Error(), "no such file or directory") {
```
Comment (ashmeenkaur) [inline tools/integration_tests/util/setup/setup.go:365]: Why was this change required?

**Review by ashmeenkaur — DISMISSED** [review]
2 nit comments, LGTM otherwise :)

File: tools/integration_tests/util/setup/setup.go (hunk 3 above)
Comment (Tulsishah) [reply tools/integration_tests/util/setup/setup.go:365]: In Delete tests of admin permissions with managed folder is already deleting `ManagedFolder1` and in teardown we again try to clean the same in case of View permission on bucket.
https://screenshot.googleplex.com/AJdfFn38mGZ2Ptp

**Review by ashmeenkaur — APPROVED** [review]
